apps/dashboard/app/(app)/logs-v2/components/controls/components/logs-display/components/display-popover.tsx

```
@@ -0,0 +1,170 @@
+  onClick: () => void;
+  isFocused: boolean;
+  index: number;
+}) => (
+  <div
+    data-item-index={index}
+    className={`font-medium text-xs p-1.5 rounded-md hover:bg-gray-4 cursor-pointer whitespace-nowrap
+      ${selected ? "bg-gray-4 text-gray-12" : "text-gray-9"}
+      ${isFocused ? "ring-2 ring-accent-7" : ""}`}
+    onClick={onClick}
+    tabIndex={isFocused ? 0 : -1}
+    role="button"
```

Biome (1.9.4)

[error] The elements with the following roles can be changed to the following elements:
<button>
<input type="button">


For examples and more information, see WAI-ARIA Roles

(lint/a11y/useSemanticElements)